docs/install/troubleshooting.md

```diff
@@ -0,0 +1,171 @@
+# Troubleshooting a Knative Installation
+
+This page describes how you can troubleshoot a Knative installation. Please try these instructions before filing a bug report.
+
+## Check if all containers are running, ready and healthy
+
+!!! note
+    The outputs below are just examples. The containers may vary depending on the installation and configuration. Please make sure all containers are running, ready and healthy (look for the `1/1` in the following examples) or are `Complete` (in case of jobs).
+
+**Knative Serving Components**
+
+```bash
+kubectl get pods -n knative-serving
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+activator-6b9dc4c9db-cl56b                1/1     Running   0          2m
+autoscaler-77f9b75856-f88qw               1/1     Running   0          2m
+controller-7dcb56fdb6-dbzrp               1/1     Running   0          2m
+domain-mapping-6bb8f95654-c575d           1/1     Running   0          2m
+domainmapping-webhook-c77dcfcfb-hg2wv     1/1     Running   0          2m
+webhook-78dc6ddddb-6868n                  1/1     Running   0          2m
+```
+
+**Knative Serving Networking Layer**
```

Comment:
nit: should we use tabs for each of the networking options, like we do on the [install page](https://github.com/knative/docs/blob/e10dc2982f7e4d7bff817c1e1a7b641247c375ec/docs/install/yaml-install/serving/install-serving-with-yaml.md#install-a-networking-layer)?

Reply:
Good idea, updated.